server/MbtaApi.py

```diff
@@ -1,10 +1,29 @@
 import secrets
 import requests
+import json_api_doc
+import Fleet
 from urllib.parse import urlencode
 
 BASE_URL_V3 = 'https://api-v3.mbta.com/{command}?{parameters}'
 
 def getV3(command, params={}, api_key=secrets.API_KEY):
     headers = {'x-api-key': api_key}
     response = requests.get(BASE_URL_V3.format(command=command, parameters=urlencode(params)), headers=headers)
-    return response.json()
\ No newline at end of file
+    return json_api_doc.parse(response.json())
+
+def transform_vehicle(vehicle):
```

Comment:
Could be inlined as a `lambda` inside `map`

Reply:
Agreed, that's cleaner; updated